app/graphql/types/solarcycle_type.rb

```diff
@@ -1,8 +1,8 @@
 
 
 class Types::SolarcycleType < Types::BaseObject
-    graphql_name "SolarCycle"
-  
-    field "type", String
-    field "time", Types::UnixDateTimeType
-  end
\ No newline at end of file
+  graphql_name "SolarCycle"
```

Comment:
Could be wrong here, but this seems to belong to another commit? Or maybe the other commit should be combined here?